### ISSUE TITLE
Relocate accumulo-proxy to separate repository on its own release schedule

### ISSUE THREAD

**Issue by ctubbsii** [apache/accumulo#454]
accumulo-proxy is really a separate project from the core Accumulo distribution, and I think it should be shipped separately, and maintained separately in its own repo.

This will also help isolate the maintenance interest, so that if there isn't any current interest, no time is wasted on it, and if there is, work can be done independently from the releases of the core Accumulo distribution.

**Comment by milleruntime** [comment]
@cjmctague thanks!

**Comment by milleruntime** [comment]
Nope sorry, this is partially done.  Still some proxy tests that need to be migrated.

**Comment by ctubbsii** [comment]
Yeah, sorry, the new repo was created, but the old proxy code is still there. It needs to be removed, and cross-referenced with the new repo, to make sure everything made it over before it is removed. I think some tests were still dependent on the proxy, maybe some minicluster features were using it also. Not sure.

**Comment by milleruntime** [comment]
@cjmctague Are you interested in working on this?

**Comment by cjmctague** [comment]
@milleruntime Yea sure. So tests need to be moved over and the current tests using the proxy need to be ported to use the new repo?

**Comment by ctubbsii** [comment]
> @milleruntime Yea sure. So tests need to be moved over and the current tests using the proxy need to be ported to use the new repo?

Tests which test the proxy should be moved over. Tests which aren't testing the proxy shouldn't be using the proxy. The proxy repo should depend on the main Accumulo repo, but not the other way around.

**Comment by hkeebler** [comment]
Working this ticket instead of 1180

**Comment by hkeebler** [comment]
Status:  Updated accumulo-proxy project from the accumulo proxy module including version updates and was able to get it to build and test runs.  Now I'll look into using the jar with the accumulo project instead of proxy module.

**Comment by hkeebler** [comment]
@ctubbsii  the accumulo-proxy contrib subdirectory has a create-release-candidate.sh script which does not exist in the accumulo project.  I'm  not sure what it does and why it is there.  Just curious.

**Comment by hkeebler** [comment]
Running a verify gave me this [ERROR] Exception is caught when Exception is not thrown in org.apache.accumulo.proxy.ProxyServer.updateRowsConditionally(String, Map) [org.apache.accumulo.proxy.ProxyServer] At ProxyServer.java:[line 2207] REC_CATCH_EXCEPTION

the code line is 
} catch (Exception e) {
      handleException(e);
      return null;
    }
This apparently verifies okay in the accumulo project so why is it failing here - I'm not savvy with verify.  I changed the catch to the following and it verifies but not sure if you have better suggestions.  
} catch (Throwable e) {
      handleException(new Exception(e));
      return null;
    }
